x/feegrant/keeper/grpc_query.go

```diff
@@ -0,0 +1,106 @@
+package keeper
+
+import (
+	"context"
+
+	"github.com/gogo/protobuf/proto"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/query"
+	"github.com/cosmos/cosmos-sdk/x/feegrant/types"
+)
+
+var _ types.QueryServer = Keeper{}
+
+// FeeAllowance returns fee granted to the grantee by the granter.
+func (q Keeper) FeeAllowance(c context.Context, req *types.QueryFeeAllowanceRequest) (*types.QueryFeeAllowanceResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
+	}
+
+	if req.Granter == "" {
+		return nil, status.Errorf(codes.InvalidArgument, "invalid granter addr")
+	}
+
+	if req.Grantee == "" {
+		return nil, status.Errorf(codes.InvalidArgument, "invalid grantee addr")
+	}
```

Comment:
```suggestion
```
We're using `AccAddressFromBech32` after which will throw an error in case of an empty address.